xml/templates/message_templates/participant_confirm_html.tpl

```diff
@@ -22,6 +22,7 @@
   <tr>
    <td>
     <p>{ts 1=$contact.display_name}Dear %1{/ts},</p>
+    <p>{ts}Your registration was previously waitlisted but you are now invited to complete the registration process.{/ts}</p>
```

Comment:
"invited to complete the registration process" feels a little odd linguistically - perhaps it's just a little too formal?.

Reply:
I can't think of anything :-p

Reply:
A. `Your registration was initially waitlisted but now we invite you to complete the registration process.`
B. `This is a confirmation that you can complete your registration that was initially waitlisted`

I think B is more in line with similar already existing texts
Examples:
`This is a confirmation that your registration has been received and your status has been updated to...`
`If space becomes available you will receive an email with a link to a web page where you can complete your registration.`

Reply:
That sounds good!